src/components/results/loadflow/load-flow-result-utils.ts

```diff
@@ -0,0 +1,191 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import {
+    LimitTypes,
+    OverloadedEquipment,
+    OverloadedEquipmentFromBack,
+} from './load-flow-result.type';
+import { IntlShape } from 'react-intl';
+import { ICellRendererParams, ValueFormatterParams } from 'ag-grid-community';
+
+const UNDEFINED_ACCEPTABLE_DURATION = Math.pow(2, 31) - 1;
+const PERMANENT_LIMIT_NAME = 'permanent';
+export const convertDuration = (duration: number): string | null => {
+    if (!duration) {
+        return null;
+    }
+
+    const minutes = Math.floor(duration / 60);
+    const seconds = duration % 60;
+
+    if (seconds === 0) {
+        return minutes + "'";
+    }
+
+    if (minutes === 0) {
+        return seconds + '"';
+    }
+    return minutes + "' " + seconds + '"';
+};
+
+export const convertSide = (side: string, intl: IntlShape) => {
+    return side === 'ONE'
+        ? intl.formatMessage({ id: 'Side1' })
+        : side === 'TWO'
+        ? intl.formatMessage({ id: 'Side2' })
+        : undefined;
+};
```

Comment:
I keeped the same logic as before. I think undefined is used on purpose because side could either have 'ONE' or 'TWO' as values.
I don't see  why I need to create enum if BRANCH_SIDE is already available, I could simply use it and keep side as string.

Reply:
I checked the api result and side could contain "" as value